src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -967,13 +967,18 @@ bool SlaterDetBuilder::readDetList(xmlNodePtr cur,
     }
 
     app_log() << "Found " << coeff.size() << " terms in the MSD expansion.\n";
+
+    if (coeff.size() == 0)
+      throw std::runtime_error(
```

Comment:
It is a uniform error and can be better handled. A lot of I/O fixes needed in this file. #5013